<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="goCheckSigOrf">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Check sig</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valSig">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Entrer le num√©ro de la SIG</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
